### PR TITLE
add argument loading

### PR DESCRIPTION
**Pull request by couleurm**
added second idea from #24 

I could've probably made it simpler by using `&str` but it involves using lifetimes and I haven't learned that yet, if you consider accepting feel free to brush off the rough corners of my addition (gave you push access to my branch).

### CODE REVIEW

File: src/vs_handler/mod.rs [hunk 1]
```diff
@@ -88,6 +90,27 @@ impl PreviewedScript {
...
+            arg.find('=')
+               .map(|index| arg.split_at(index))
+               .map(|(k, v)| (k, &v[1..]))
+               .ok_or_else(|| anyhow!("No value specified for argument: {}", arg))
+        }
+
+        // https://github.com/YaLTeR/vapoursynth-rs/blob/60f2656ed76a3f5dc40266618a82d80f7b49fbd6/vapoursynth/examples/vspipe.rs#L722
+        if let Some(args) = &self.args {
+            let mut args_map = OwnedMap::new(API::get().unwrap());
+            for arg in args.iter().map(|s| &**s).map( parse_arg) {
```
Comment (quietvoid) [inline src/vs_handler/mod.rs:103]: This mapping seems unnecessary, there exists a [split](https://doc.rust-lang.org/std/primitive.str.html#method.split) function that's more idiomatic.

I'll comment a better version when I have time later.
`parse_arg` could be completely removed and there's no need to add `.iter()` over a `Vec`.

**Comment by quietvoid** [comment]
@couleurm Can you please test the changes.
The variables can be set with multiple `-v` args or a list delimited by `,`.

```console
vspreview-rs -v key=value -v ...
vspreview-rs -v key1=value1,key2=value2,...
```

**Comment by couleurm** [comment]
hey, it'd be cool to stick to VSPipe's conventions and add support for `-a` and `--arg`, and add some syntatic sugar by separating with them `,` to avoir redundancy, good call

this would make it much more convenient to switch back and from vs pipe and preview-rs, (i'm the prolly the only one who use them that much)

**Comment by quietvoid** [comment]
There's an alias now.

**Pull request merged**